src/styles/index.scss

```diff
@@ -1,3 +1,60 @@
+@import 'variables';
+
 body {
   margin: 0;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: $black-color;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $border-day-gap;
+
+  padding: 10px;
+  width: ($day-size * 7 + $border-day-gap * 6);
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid $black-color;
```

Comment:
The variable `$black-color` should be defined in the `variables.scss` file. Ensure that it is correctly imported and defined there.